### DISCUSSION TITLE
Problem using Focusrite 4i4 with Jamulus

### DISCUSSION THREAD

**Discussion by TheFogster** [jamulussoftware/jamulus#3245] [Technical Support & Help]
Hello Everyone I'm hoping someone will be able to help me.
I have a focusrite 4i4 4th generation which I am trying to use with the latest version of Jamulus. I am quite an experienced user and I am using a Mac with latest OS. The 4i4 works fine with my DAW (Logic) however when I try to use it with Jamulus I cannot hear any sound from my mic or guitar through my headphones, even though I can hear the other musicians I am playing with and they can hear me. I can also see the signal is sending on the front panel of the interface and in Focusrite control and also In the Jamulus control window. It just isn't getting to my headphones. I'm baffled and hoping for a solution. Thanks for listening

**Comment by chrisrimple** [comment]
Let's confirm a few things:

1. You're sending sound to the Jamulus server.  Appears to be YES, since other musicians can hear you.
2. You're receiving sound from the Jamulus server.  Appears to be YES, since you hear other musicians.
3. You're hearing yourself in the mix from the Jamulus server.  Appears to be NO.

When you look at the mixer window in the Jamulus client, are you Muted?  There are 2 mute buttons, and they function differently.  "Mute Myself" stops sending your sound to the server, but continues to let you hear it locally.  "Mute" under your volume slider blocks your sound that's coming back from the server.  So my guess is that you've accidentally clicked ON the Mute under your volume slider.

Are you using the Solo buttons under any volume slider?  If so, you'll only hear the server sound for the musicians with Solo ON.  Maybe you have a bunch of musicians with Solo ON, but you don't have Solo ON under your own volume slider?

Apart from those two, I don't know what else could cause the problem you're describing.

**Reply by TheFogster** [reply]
Thanks for getting back to me Chris. 1,2 and 3 are correct. I understand the "mute myself" button and it is off. In the past I was always told to turn on "mute" under the slider otherwise there is a lot of delay when playing so that is on and that's how I've worked over the past years since lockdown. Currently I'm writing with only one other musician. I recently upgraded to the Focusrite 4i4 from a 2i2 which fortunately I haven't sold, so this morning I transferred over to the 2i2 and it worked just fine. So it seems to be an issue with the 4i4 and Jamulus because it works perfectly with Logic. I remain baffled but I thank you for your kind and prompt reply. I may have to contact Focusrite direct if that is possible.
Cheers
John

**Reply by chrisrimple** [reply]
If you Mute your volume slider, you're not hearing yourself from the Jamulus server.  Does that mean that you've not been hearing yourself in the past?  It sounds like you have been.

Then you say that your 2i2 "works" and the 4i4 does not.  Which makes me guess:

- Using your 2i2, you set Mute ON for your volume slider, so were never hearing your sound from the Jamulus server
- Using your 2i2, you enabled (whether you realized it or not) "local monitoring", meaning that the audio interface is sending your sound back to your headphones as you play it (whether it goes to the Jamulus server or not)
- Using your 4i4, you do not have "local monitoring" enabled, so you're no longer hearing yourself

If I'm right, there's nothing wrong with your 4i4.  But also, you're using Jamulus incorrectly. You should NEVER listen to "local monitoring", and instead should Mute OFF your volume slider, so that you're ONLY listening to yourself in the sound returned from the Jamulus server.  If you're listening to "local monitoring", it's much harder to stay in sync with the other musicians, since they're all listening to the Jamulus server audio.

**Reply by TheFogster** [reply]
Thanks again Chris I will investigate further and hopefully find the solution.Cheers

**Comment by henkdegroot** [comment]
I noticed the 4i4 has a control app which allows to setup the signal routing in the focusrite. Maybe some setting has been changed? Or perhaps the default setting is not what you expect it to be.

See https://support.focusrite.com/hc/en-gb/articles/360006972599-Direct-Monitoring-on-the-Scarlett-4i4-3rd-Gen

**Reply by henkdegroot** [reply]
And maybe some hints in this video as well regarding the setup: https://www.youtube.com/watch?v=PxtDIadIVfI

**Reply by TheFogster** [reply]
Thanks for your reply and the links, much appreciated
